x-pack/plugin/esql/qa/server/multi-node/build.gradle

```diff
@@ -1,11 +1,27 @@
 apply plugin: 'elasticsearch.internal-java-rest-test'
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
 
 dependencies {
   javaRestTestImplementation project(xpackModule('esql:qa:testFixtures'))
   javaRestTestImplementation project(xpackModule('esql:qa:server'))
+  yamlRestTestImplementation project(xpackModule('esql:qa:server'))
 }
 
 
 tasks.named('javaRestTest') {
   usesDefaultDistribution()
 }
+
+restResources {
+  restApi {
+    include '_common', 'bulk', 'get', 'indices', 'esql', 'xpack', 'enrich', 'cluster'
+  }
+  restTests {
+    includeXpack 'esql'
+  }
+}
+
+tasks.named('yamlRestTest') {
+  usesDefaultDistribution()
+  maxParallelForks = 1
```

Comment:
This shouldn't really be necessary. There's only one test suite in this source set anyway.